hudi-spark-datasource/hudi-spark-common/src/main/java/org/apache/hudi/commit/DatasetBucketRescaleCommitActionExecutor.java

```diff
@@ -73,10 +66,4 @@ protected void preExecute() {
     ValidationUtils.checkArgument(res);
     LOG.info("Finish to save hashing config " + hashingConfig);
   }
-
-  @Override
-  protected Map<String, List<String>> getPartitionToReplacedFileIds(HoodieData<WriteStatus> writeStatuses) {
```

Comment:
we still need to overwrite this `getPartitionToReplacedFileIds`  in DatasetBucketRescaleCommitActionExecutor ，Get rid of the influence of `String staticOverwritePartition = config.getStringOrDefault(HoodieInternalConfig.STATIC_OVERWRITE_PARTITION_PATHS); `parameter